scm/doc/TechGuide/chap_hsd.tex

```diff
@@ -0,0 +1,212 @@
+This will create the conda environment \execout{scm\_suite\_sim}
+
+
+Activate environment:
+
+\execout{> conda activate scm\_suite\_sim}
+
+\subsection{Enabling the CCPP Suite Simulator}
+
+To use the CSS in the CCPP-SCM two modifications need to be made:
+
+\begin{enumerate}
+\item Add CSS, and any interstitial schemes needed for coupling the CSS to the host (e.g. SCM), to an existing CCPP SDF (or create a new SDF).
+\item Modify, or create new, namelist that has the options needed to activate the CSS.
+\end{enumerate}
+
+\subsection{Modifying the CCPP Suite Definition File}
+
+The SDF needs to be modified to include the CSS scheme, and any interstitial schemes needed for your host application. Here we will illustrate how to use the CSS within SCM (UFS) physics, for which all applications use a physics SDF with mixture of process-split and time-split physics schemes.
+In general,
+\begin{itemize}
+\item for SDFs that contain ONLY process-split schemes, the CSS can be added to the end of the SDF for all configurations. In this case we have the flexibility to switch ``on/off'' any combination of active physics and simulated physics at runtime, via namelist, with the same modified SDF.
+\item when using SDFs that contain ONLY time-split schemes, the CSS needs to be added before and after each scheme you want to switch ``on/off''. So one could add calls to the CSS between each process in the SDF to obtain total flexibility, or just around the schemes they are interested in.
+\end{itemize}
+
+In the examples below we will demonstrate how to modify SDFs to use the CSS for SCM (UFS) physics applications, \ref{section:Suite_with_Active_Radiation} and \ref{section:Suite_with_Active_Cloud_Microphysics}.
+
+\subsection{Namelist for the CCPP Suite Simulator}
+
+The CSS has its own namelist, \execout{ccpp\_suite\_sim\_nml}, that needs to be added to the physics namelists used by the SCM.
+
+\lstinputlisting[
+                 basicstyle=\scriptsize\ttfamily,
+                 label=lst_css_nml,
+                 caption=Example namelist for CCPP Suite Simulator.
+                 ]{./css_nml.txt}
+
+\begin{itemize}
+\item \execout{suite\_sim\_file}: Input file with simulated data tendencies (See \ref{section:Creating_Custom_Data_for_Simulator} for how to create input file from SCM output).
+\item \execout{nprc\_sim}: Number of physical processes in the input data.
+\item \execout{prc\_XYZ\_cfg}: Configuration for physical process XYZ.
+\begin{itemize}
+\item 0 - Active scheme; 1 - Use data
+\item 0 - Process-split scheme; 1 - Time-split scheme
+\item Index for scheme order (1 - \execout{nprc\_sim})
+\end{itemize}
+\end{itemize}
+
+For example, in \ref{lst_css_nml}, there are two active schemes, longwave and shortwave radiaiton, and five simulated schemes: PBL, gravity-wave drag, deep/shallow convection, and cloud microphysics. The radiation, gravity-wave drag and PBL schemes are all process-split, whereas convection and cloud microphysics are time-split. 
```

Comment:
This doesn't match what I see in 7.2 where I see only PBL active or PBL and radiation active